core/server/data/migrations/versions/4.35/2022-01-30-15-17-set-welcome-page-url-from-settings.js

```diff
@@ -0,0 +1,39 @@
+const logging = require('@tryghost/logging');
+
+// For DML - data changes
+const {createTransactionalMigration} = require('../../utils');
+
+module.exports = createTransactionalMigration(
+    async function up(knex) {
+        logging.info('Setting welcome_page_url from settings');
+
+        const paidSignupRedirect = await knex()
+            .select('value')
+            .from('settings')
+            .where('key', 'members_paid_signup_redirect')
+            .first();
+
+        const freeSignupRedirect = await knex()
+            .select('value')
+            .from('settings')
+            .where('key', 'members_free_signup_redirect')
+            .first();
+
```

Comment:
Can we check for the existence of `paidSignupRedirect` and `freeSignupRedirect` here? They __should__ exist but I'd like to prevent migrations crashing if not

Reply:
Sure thing - I'll exit early if they're not present!

Reply:
I decided to wrap the inserts with checks - I think it's better because we may have just one